scripts/updateReleaseToLocalDev.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Use this script after pulling down a local copy of the AWS Swift SDK to your local machine.
+# Doing so will allow you to build (and modify) a local copy of the AWS Swift SDK.
+#
+# After setting this up, you can update your executable's Package.swift to depend on a local version of the SDK via:
+#
+#    dependencies: [
+#        .package(name: "AWSSwiftSDK", path:"~/Projects/SwiftSDK_Releases/aws-sdk-swift"),
+#    ],
+
+#
+# Setup Instructions
+#  Note: Versions listed below should be updated to your preference (ideally, the latest)
+#
+# mkdir -p ~/Projects/SwiftSDK_Releases
+# cd ~/Projects/SwiftSDK_Releases
+#
+# git clone git@github.com:awslabs/aws-sdk-swift.git
+# cd aws-sdk-swift
+# git checkout -b 0.1.1
+# cd ..
+#
+# git clone git@github.com:awslabs/smithy-swift.git
+# cd smithy-swift
+# git checkout -b 0.1.1
+# cd ..
+#
+# git clone --recurse-submodules git@github.com:awslabs/aws-crt-swift.git
+# cd aws-crt-swift
+# git checkout -b 0.1.0
+# cd ..
+#
+# ./aws-sdk-swift/scripts/updateReleaseToLocalDev.sh
+#
+#  If this script runs successfully, you can depend on a local version of the SDK
+
+
+AWS_SDK_SWIFT=
+SMITHY_SWIFT=
+AWS_CRT_SWIFT=
+
+determineAbsolutePath() {
+    SCRIPTDIR=`dirname $1`
+    pushd ${SCRIPTDIR} > /dev/null
+    cd ../../
+    AWS_SDK_SWIFT=$PWD/aws-sdk-swift
+    SMITHY_SWIFT=$PWD/smithy-swift
+    AWS_CRT_SWIFT=$PWD/aws-crt-swift
+}
+
+determineAbsolutePath $0
+
+if [ -z ${AWS_SDK_SWIFT} ] || [ ! -d ${AWS_SDK_SWIFT} ]; then
```

Comment:
is this checking for null or empty path here?

Reply:
We are checking if the string is empty -- if it is empty, we bail.. if it isn't, then we check to make sure that the directory is there.  If the directory is not there, then we bail.